src/renderer/contexts/DependencyContext.tsx

```diff
@@ -458,12 +464,38 @@ export const DependencyProvider = memo(({ children }: React.PropsWithChildren<un
                                     Python ({pythonInfo.version}) [
                                     {isSystemPython ? 'System' : 'Integrated'}]
                                 </Text>
-                                <IconButton
-                                    aria-label="Open Console View"
-                                    icon={<BsTerminalFill />}
-                                    size="sm"
-                                    onClick={() => setIsConsoleOpen(!isConsoleOpen)}
-                                />
+                                <HStack>
+                                    <HStack>
+                                        <Switch
+                                            isChecked={usePipDirectly}
```

Comment:
We should disable the switch while (un)installing deps.

```suggestion
                                            isChecked={usePipDirectly}
                                            isDisabled={isRunningShell}
```